### ISSUE TITLE
Remove deprecated unzip and unzipIter in twisted.python.zipstream

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5766]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#5766|
|Type|task|
|Created|2012-07-11 10:25:23Z|
|Branch|https://github.com/twisted/twisted/tree/remove-unzip-5766|

`unzip` and `unzipIter` in twisted.python.zipstream were deprecated in Twisted 11.0 and can be removed.

Same goes for `countZipFileEntries`, deprecated in r22464.

Attachments:

* [zipstream-5766.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/05e/05eb915fbebb5cae2380562f739203d2a1594936/f85bd43c61d71a228c619991c94a4a219a6112e9.patch) (15478 bytes) - added by thijs on 2012-07-11 11:33:33Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5766 5766
type__task task
reporter__thijs thijs
priority__low low
milestone__ 
branch__branches_remove_unzip_5766 branches/remove-unzip-5766
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1342002323000000 1342002323000000
changetime__1344457035000000 1344457035000000
version__None None
owner__thijs thijs
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

The capitalization and punctuation changes in this patch are questionable.  If they came with improvements to the actual content of the documentation itself, I'd be happy to accept them.  Twiddling with periods for its own sake strikes me as a great waste of time though.

Please put this patch in a branch and verify that it passes on all builders.  If it does, go ahead and merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

(In [35093]) Branching to 'remove-unzip-5766'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra commented|
|-|-|

(In [35094]) apply zipstream-5766.patch without punctuation, refs [#5766](https://github.com/twisted/twisted/issues/5766)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set status to `assigned`|
|-|-|

Thanks for the review. The patch was committed without the punctuation changes: [Build results](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/remove-unzip-5766).

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [35095]) Merge remove-unzip-5766: unzip, unzipIter, and countZipFileEntries in twisted.python.zipstream (deprecated in Twisted 11.0) are removed now.

Author: thijs
Reviewer: exarkun
Fixes: [#5766](https://github.com/twisted/twisted/issues/5766)
